module/VuFind/tests/unit-tests/src/VuFindTest/Config/PathResolverTest.php

```diff
@@ -0,0 +1,157 @@
+<?php
+/**
+ * Config Path Resolver Test Class
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2022.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+namespace VuFindTest\Config;
+
+use VuFind\Config\Locator;
+use VuFind\Config\PathResolver;
+use VuFindTest\Feature\FixtureTrait;
+use VuFindTest\Feature\PathResolverTrait;
+
+/**
+ * Config Path Resolver Test Class
+ *
+ * @category VuFind
+ * @package  Tests
+ * @author   Ere Maijala <ere.maijala@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:testing:unit_tests Wiki
+ */
+class PathResolverTest extends \PHPUnit\Framework\TestCase
+{
+    use FixtureTrait;
+    use PathResolverTrait;
+
+    /**
+     * Stacked path resolver
+     *
+     * @var PathResolver
+     */
+    protected $stackedResolver;
+
+    /**
+     * Setup method.
+     *
+     * @return void
+     */
+    public function setUp(): void
+    {
+        $fixtureDir = $this->getStackedFixtureDir();
+        $this->stackedResolver = new PathResolver(
+            APPLICATION_PATH,
+            [
+                $fixtureDir . 'middle',
+                $fixtureDir . 'bottom',
```

Comment:
I'm a little confused by this test, because the "bottom" of the stack based on the file name is actually what I would consider the "top" of the stack. Am I missing something?

Reply:
Ended up renaming the fixtures.